modules/core/src/main/java/org/apache/ignite/internal/LongJVMPauseDetector.java

```diff
@@ -117,6 +126,13 @@ public void start() {
                                 longPausesTimestamps[next] = now;
 
                                 longPausesDurations[next] = pause;
+
+                                lastWakeUpTime = now;
```

Comment:
i hope u need sync sections in all lastWakeUpTime usage or better replace with Atomic?